packages/atomic/src/pages/examples/ipx.html

```diff
@@ -0,0 +1,157 @@
+          <atomic-breadbox></atomic-breadbox>
+          <atomic-query-summary></atomic-query-summary>
+          <atomic-did-you-mean></atomic-did-you-mean>
+          <atomic-notifications></atomic-notifications>
+        </atomic-layout-section>
+        <atomic-layout-section section="results">
+          <atomic-smart-snippet></atomic-smart-snippet>
+          <atomic-smart-snippet-suggestions></atomic-smart-snippet-suggestions>
+          <atomic-result-list image-size="none">
+            <atomic-result-template>
+              <template>
+                <style>
+                  .field {
+                    display: inline-flex;
+                    align-items: center;
+                  }
+
+                  .field-label {
+                    font-weight: bold;
+                    margin-right: 0.25rem;
+                  }
+
+                  .thumbnail {
+                    display: none;
+                    width: 100%;
+                    height: 100%;
+                  }
+
+                  .icon {
+                    display: none;
+                  }
+
+                  .result-root.image-small .thumbnail,
+                  .result-root.image-large .thumbnail {
+                    display: inline-block;
+                  }
+
+                  .result-root.image-icon .icon {
+                    display: inline-block;
+                  }
+
+                  .result-root.image-small atomic-result-section-visual,
+                  .result-root.image-large atomic-result-section-visual {
+                    border-radius: var(--atomic-border-radius-xl);
+                  }
+                </style>
+                <atomic-result-section-badges>
+                  <atomic-field-condition must-match-sourcetype="Salesforce">
+                    <atomic-result-badge label="Salesforce" class="salesforce-badge"></atomic-result-badge>
+                  </atomic-field-condition>
+                  <atomic-result-badge
+                    icon="https://raw.githubusercontent.com/Rush/Font-Awesome-SVG-PNG/master/black/svg/language.svg"
+                  >
+                    <atomic-result-multi-value-text field="language"></atomic-result-multi-value-text>
+                  </atomic-result-badge>
+                  <atomic-field-condition must-match-is-recommendation="true">
+                    <atomic-result-badge label="Recommended"></atomic-result-badge>
+                  </atomic-field-condition>
+                  <atomic-field-condition must-match-is-top-result="true">
+                    <atomic-result-badge label="Top Result"></atomic-result-badge>
+                  </atomic-field-condition>
+                </atomic-result-section-badges>
+                <atomic-result-section-title><atomic-result-link></atomic-result-link></atomic-result-section-title>
+                <atomic-result-section-title-metadata>
+                  <atomic-field-condition class="field" if-defined="snrating">
```

Comment:
C'est voulu que tu ajoutes des fields de ServiceNow ? Tu pourrais mettre des fields plus génériques sinon 😄

Reply:
Y'en a un de Salesforce en haut. C'est vraiment juste des exemples génériques faq ça a pas d'incidences. J'ai copié ça [des templates de Insight](https://github.com/coveo/ui-kit/blob/feature/SVCINT-1915-ipx-in-atomic/packages/atomic/src/pages/examples/insights.html#L137)